spec/bolt/node/ssh_spec.rb

```diff
@@ -15,6 +15,10 @@
   let(:port) { 2224 }
   let(:command) { "pwd" }
   let(:ssh) { Bolt::SSH.new(hostname, port, user, password) }
+  let(:key) {
+    { config: Bolt::Config.new(key: Dir[".vagrant/**/private_key"],
+                               insecure: true) }
+  }
```

Comment:
It looks like this depends on an external file, so probably won't work for anyone else? Note we don't run tests marked with `vagrant: true` in appveyor or travis, but only when running `bundle exec rake test`.

Reply:
This file is created under the .vagrant directory when you play `vagrant up` before launching `bundle exec rake test`. Unless I am mistaken, `bundle exec rake test` run some tests with vagrant in contrast to `bundle exec rake unit`.